api/envoy/config/core/v3/proxy_protocol.proto

```diff
@@ -47,4 +56,10 @@ message ProxyProtocolConfig {
   // This config controls which TLVs can be passed to upstream if it is Proxy Protocol
   // V2 header. If there is no setting for this field, no TLVs will be passed through.
   ProxyProtocolPassThroughTLVs pass_through_tlvs = 2;
+
+  // Custom TLV metadata to be sent in the upstream PROXY protocol header. This field
+  // is only valid when the version is V2. These TLVs can be defined both in the
+  // endpoint host's typed metadata and at the config-level. In the case that the same
```

Comment:
Please document the namespace they would go in for the host-level data, and maybe make an inline config example of specifying it on a host in the docs here.

Reply:
Done in https://github.com/envoyproxy/envoy/pull/37591/commits/e50c959cf46fc35e54faaff24963f931944ddfb4. Resolving to consolidate the proto-related changes to the https://github.com/envoyproxy/envoy/pull/37591#discussion_r1906046575 review comment.